### DISCUSSION TITLE
Is there a way to extract Subject-Verb-Object triplets?

### DISCUSSION THREAD

**Discussion by santoshbs** [explosion/spaCy#6280] [Help: Coding & Implementations] [answered]
Using spacy, is there a way to obtain all SVO triplets from a corpus?

**Comment by svlandeg** [comment, marked as answer]
If you have a model with a trained parser, you can [navigate the dependency tree](https://spacy.io/usage/linguistic-features#navigating) looking for any type of patterns you want.

Additionally, spaCy 3 features a new [`DependencyMatcher`](https://nightly.spacy.io/usage/rule-based-matching#dependencymatcher) that allows you to match patterns within the dependency parse. There is currently a pre-release of spaCy v3 available as [`spacy-nightly`](https://pypi.org/project/spacy-nightly/), you can find more information here: https://explosion.ai/blog/spacy-v3-nightly.

If you end up playing with the DependencyMatcher, we'd love to hear your feedback!
